app/controllers/posts_controller.rb

```diff
@@ -33,6 +33,23 @@ def create
     end
   end
 
+  def destroy
```

Comment:
Well, you made a great effort for making this project. However there are some places need to be fixed. 

- [ ] In the requirement you need to load authorization like that. But I cannot see it in your controllers:


```ruby
class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Kindly fix this issue.

Reply:
resolved